geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -92,16 +90,12 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType
         logger.debug(s"Creating table $name")
         val descriptor = new HTableDescriptor(name)
         descriptor.addFamily(HBaseFeatureIndex.DataColumnFamilyDescriptor)
-        Option(admin.getConfiguration.get(CoprocessorHost.USER_REGION_COPROCESSOR_CONF_KEY)) match {
+        if (ds.config.remoteFilter) {
```

Comment:
What's the default for remote filtering?  I'm wondering if the command line tools will default to setting up the coprocessors or not...

Reply:
default is true. it checks the ds param first, then the system prop, then defaults to true